magento-research/pwa-studio
# [dev]: Update feather to 2.0.x

## Description

Updates feathericons to 2.0.3 and uses named imports instead of specifying path.

## Related Issue





Closes #1415 .

## Verification Steps

1. General regression test of the app to make sure icons appear and display properly.
2. In `venia-concept` run `yarn build:analyze` before and after this PR. Compare size. After this PR we should gain less than a KB in size (probably due to updates in the feather repo).

## Screenshots / Screen Captures (if appropriate)
Before
[![Image from Gyazo](https://i.gyazo.com/8c573bca0c5d29ca45519e5501b8cd68.png)](https://gyazo.com/8c573bca0c5d29ca45519e5501b8cd68)

After
[![Image from Gyazo](https://i.gyazo.com/5a2f9120efce5ae3683a46838fd18edf.png)](https://gyazo.com/5a2f9120efce5ae3683a46838fd18edf)
## Proposed Labels for Change Type/Package




## Checklist:
